news/2019-01-30-new-changes-to-star-rating-performance-points.md

```diff
@@ -0,0 +1,81 @@
+
+Decided not to farm a map because it had square jumps? Found yourself baffled by the difficulty of an older 2013 map with linear flow? These previously underweighted maps now have a more accurate star rating associated with them (this includes alt maps). And yes, they will be worth more PP as a result.
+
+## Streams and High BPM
+
+Besides the changes to aim, there has been an adjustment to how streams are handled by the algorithm. Many players who are capable of incredible speeds find their ranks are unfairly low given their unique skillset. To promote these unique skillsets, streams exceeding 200 BPM now have an increased reward that more accurately represents their difficulty.
+
+Able to FC 300 BPM stacks with your laptop keyboard? This is the change you've been waiting for. 
+
+In addition, a small nerf to very spaced streams has been introduced. These are those streams found in maps such as Honesty and Sidetracked Day. While the nerf is sizable, these plays are still quite impressive and are still worth a lot, just not quite as much as they were worth before.
+
+## Slider Buffs
+
+Are you a fan of playing those impressive and previously underweighed tech maps with high slider velocity? 
+
+Well, say no more! 
+
+While the specifics are quite complicated, in general, these fast sliders are now more accurately judged by the algorithm. Not only that, but maps with long sliders that have jumps between them also receive a large buff compared to before.
+
+This is a change that will help buff many tech maps with fast gimmicky sliders. While this change will not be noticeable on many maps, the ones that do benefit may see substantial buffs!
+
+## Flashlight Rework
+
+One of the exciting introductions that will be coming with this patch is the change to flashlight. Primarily a dead mod outside of farming 30 second maps, flashlight has been irrelevant since PPV2's inception back in 2014. 
+
+To remedy this, flashlight will now scale at a much higher rate with the length of a map. To compensate for this increased buff for longer maps, the shorter maps that were previously played with flashlight will receive a small nerf.
+
+## Hidden Rework
+
+Yes, HD is changing again. With the introduction of the high BPM buff, we wanted to make changes to HD so that maps with fast stacked streams weren't overly buffed when HD is applied. 
+
+Instead of the old system, HD now rewards accuracy more, and both aim and speed PP will scale off of the approach rate of the map. 
+
+Find yourself good at farming EZHD plays? You'll find a notable increase in performance points for these plays after the update. 
+
+## Misc PP Changes
+
+These are the more minor technical changes that we introduced to keep things balanced. 
+
+To combat potentially unimpressive keyboard mashing that may be common for high speed plays as a result of the new buff, a new reward curve for accuracy has been introduced for the speed skill.
```

Comment:
```suggestion
To combat potentially unimpressive keyboard mashing that may be common for high speed plays as a result of the new buffs, a new reward curve for accuracy has been introduced for the speed skill.
```

Reply:
Also differentiating between buffs and nerfs seems wrong. IMO replace every instance with "changes".